word_counter.rb

```diff
@@ -0,0 +1,28 @@
+class WordCounter
+  attr_reader :filename, :word_hash
+
+  def initialize(filename)
+    @filename = filename
+    @word_hash = generate_sorted_word_hash
+  end
+
+  def count
```

Comment:
One thing I'd think about is this method name here. It basically promises to count the words, but actually prints out a count of the words. I think a name like print_word_counts would be more descriptive of what this is doing.